src/components/home/AllCollections.js

```diff
@@ -304,8 +339,11 @@ function AllCollections() {
               />
             </>
           )}
-          {(selectedOptions[0] || selectedOptions[1]) && (
-            <LevelCollectionsPage selectedOptions={selectedOptions} />
+          {(selectedLevel?.length > 0 || selectedDiscipline?.length > 0) && (
```

Comment:
If I'm not mistaken, line 342 is the opposite of line 324. Perhaps you could juste use `condition ? x : y` instead of having both conditions.